### PR TITLE
Copy update one

### PR DESCRIPTION
**Pull request by jesusxambro**

### CODE REVIEW

**Review by markanator — APPROVED** [review]

File: src/components/Services.astro [hunk 1]
```diff
@@ -128,6 +129,7 @@
 </section>
 
 <script>
+  console.log('Swiper initialized!')
```
Comment (markanator) [inline src/components/Services.astro:132]: stray log

File: src/components/Test.astro
Comment (markanator) [inline src/components/Test.astro:1]: is this file needed?

Comment (jesusxambro) [reply src/components/Test.astro:1]: No, I was trying to get some slick thing going but it didn't work.
